src/index.html

```diff
@@ -7,62 +7,694 @@
     <title>Way to Middle</title>
   </head>
   <body>
-    <!--    <h1>hello</h1>-->
-    <!--    <img src="assets/webpack-logo.png" />-->
-    <label class="checkbox">
-      <input type="checkbox" class="checkbox__input" />
-      <span class="checkbox__decor">
-        <svg class="checkbox__decor-icon" fill="#fff" width="12" height="12">
-          <use xlink:href="#plus"></use>
-        </svg>
-      </span>
-    </label>
-
-    <div class="article">
-      <h1 class="article__title"></h1>
-      <a href="#" class="see-more-action">
-        See more
-        <svg class="see-more-action__icon" width="25" height="25">
+    <div class="container">
+<!-- ==================== CARD 1 ========================== -->
```

Comment:
[minor]

A little roughness with the comment align. 
Tab it to make the same as on the line 173

Do the same on the line 82